src/test/java/org/sonar/plugins/findbugs/FindbugsSensorTest.java

```diff
@@ -185,7 +182,7 @@ public void should_execute_findbugs_even_if_only_findsecbug() throws Exception {
   }
 
   @Test
-  public void should_execute_findbugs_but_not_find_violation() throws Exception {
+  public void should_not_execute_findbugs_if_no_active() throws Exception {
 
     BugInstance bugInstance = getBugInstance("THIS_RULE_DOES_NOT_EXIST", 107);
```

Comment:
I think two different tests will be required. This test initially tested that bug instance with unknown id (for example a bug report that is not part of the Sonar descriptions or not supported).